AUTHORS.rst

```diff
@@ -1,5 +1,5 @@
 Authors
-=======
+====================
```

Comment:
Please check whether this breaks the compiled build and report back....I thought that the number of underscores/whatever had to match the number of letters, but I am not sure.

Reply:
In the pycharm preview it looks fine which I believe shows the compiled version and elsewhere we have an entire row of "=" for smaller titles
<img width="629" alt="Screen Shot 2022-12-01 at 3 56 56 PM" src="https://user-images.githubusercontent.com/68021211/205157728-cd6fa709-5be5-4a3f-a822-a8152df86b45.png">